opendbc/safety/safety/safety_hyundai_canfd.h

```diff
@@ -241,6 +281,7 @@ static int hyundai_canfd_fwd_hook(int bus_num, int addr) {
 static safety_config hyundai_canfd_init(uint16_t param) {
   const int HYUNDAI_PARAM_CANFD_LKA_STEERING_ALT = 128;
   const int HYUNDAI_PARAM_CANFD_ALT_BUTTONS = 32;
+  const int HYUNDAI_PARAM_CANFD_ANGLE_STEERING = 256;
```

Comment:
@sunnyhaibin this is the wrong flag, this is for FCEV. You have to be extremely careful when writing panda safety

Reply:
None of the MISRA/safety tests are passing either, it's important to--regardless of safety--close the loop on the tests. It's okay if you need help, just communicate where you need help!